packages/jest-puppeteer-axe/src/index.ts

```diff
@@ -54,31 +50,44 @@ function formatViolations( violations ) {
 		.join( '\n' );
 }
 
+interface AxeTestParams {
+	include?: string | string[];
+	exclude?: string | string[];
+	disabledRules?: string[];
+	options?: RunOptions;
+	config?: Spec;
+}
+
+interface MatcherContext {
+	utils: {
+		matcherHint: ( matcherName: string ) => string;
+		RECEIVED_COLOR: ( text: string ) => string;
+	};
+}
```

Comment:
We don't need this interface here. It's already available globally via `jest.MatcherContext`